Bio/Align/_pairwise.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2000, 2004 by Brad Chapman.
```

Comment:
Although the classes here were in ``Bio/Align/__init__.py`` I don't think that copyright block applies.

Rather it looks like your name should be here - it was added in https://github.com/biopython/biopython/pull/1655